### PR TITLE
VIH-6860 set call tag before calling pexip

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-6860

### Change description ###

* set call tag before calling pexip
* update pexip client
* simplify pexip to dto mapping

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by valentinabennett — APPROVED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/scripts/pexrtc.js [hunk 1]
```diff
@@ -3568,9 +3575,9 @@ PexRTC.prototype.getPresentationURL = function() {
         if (self.png_presentation) {
             url = "https://" + self.node + "/api/client/v2/conferences/" + self.conference_uri + "/presentation.png?id=" + self.presentation_event_id + "&token=" + self.token;
         } else {
-	    if (self.bandwidth_in > 512) {
-		presentation_image = "presentation_high.jpeg";
-	    }
+            if (self.bandwidth_in > 384) {
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/scripts/pexrtc.js:3578]: does this need to be configurable? where did 384 come from? :)

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts [hunk 2]
```diff
@@ -98,6 +104,7 @@ export class VideoCallService {
      * @param maxBandwidth the maximum bandwith
      */
     makeCall(pexipNode: string, conferenceAlias: string, participantDisplayName: string, maxBandwidth: number) {
+        this.initCallTag();
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts:107]: does setting in make call meet this AC? 
"value should be unique every time the user disconnects and re-connects to pexip"

File: VideoWeb/VideoWeb/ClientApp/src/scripts/pexrtc.js (hunk 1 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/scripts/pexrtc.js:3578]: This is the Pexip client code. I believe we do set the bandwidth in our component.
See `WaitingRoomBaseComponent` line 36 where to set the bandwidth to a constant value.

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts (hunk 2 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts:107]: This has to be set before make call. In the components, we call makeCall on unexpected disconnects

**Review by RuwanikaWeerakkody — APPROVED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/video-call-models.spec.ts [hunk 3]
```diff
@@ -1,43 +1,84 @@
...
 describe('ParticipantUpdated', () => {
     const participantDisplayName = new ConferenceTestData().getConferenceDetailNow().participants[0].tiled_display_name;
-    let participantUpdated: ParticipantUpdated;
-    let conferenceUpdated: ConferenceUpdated;
+    const testData = new VideoCallTestData();
 
     it('should return muted status', () => {
-        participantUpdated = new ParticipantUpdated('YES', 1234, participantDisplayName, Guid.create().toString(), 0);
+        const pexipParticipant = testData.getExamplePexipParticipant(participantDisplayName);
+        pexipParticipant.is_muted = 'Yes';
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/video-call-models.spec.ts:11]: others are YES NO - all caps

Comment (levibutler) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/video-call-models.spec.ts:11]: oops nope i hadnt scrolled down enough there is a good mix

**Review by levibutler — APPROVED** [review]

**Pull request merged**